pytorch/inference/docker/1.12/py3/cu116/Dockerfile.gpu

```diff
@@ -215,6 +215,10 @@ RUN pip install --no-cache-dir -U \
  && pip uninstall -y torch torchvision torchaudio torchdata \
  && pip install --no-cache-dir -U ${PT_INFERENCE_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
 
```

Comment:
you are still installing cryptography at line 207 where you removed it on the other dockerfiles. Think you want to remove here as well.

Reply:
Good catch.

Reply:
This a different file. The one you saw was in cu113/Dockerfule.gpu

Reply:
the process you used on the other 2 dockerfiles was removing cryptography from the main pip install and added below. In this case, you have cryptography being installed twice which not best practice and not inline with the previous changes mades. Please remove from the upper block like you did in the other dockerfiles.